### DISCUSSION TITLE
Testing Idena node: release candidate 1.0.2-rc1

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-go#1114] [General]
We invite you to test the release candidate version of the Idena node: `1.0.2-rc1`
To test it please download [rc-version of the Idena node](https://github.com/idena-network/idena-go/releases/tag/v1.0.2-rc1) manually.

Changes:

- Supporting SSE4.2 CPUs without AVX instructions

We plan to release these changes in the next update unless negative feedback is received.

**Comment by ghost** [comment]
I download it, I don't have a problem for now. Good job. If I don't comment in few hour, that means it's working !
